scylladb/sphinx-scylladb-theme
# Add hide_versions_dropdown option

Adds a new option to completely disable the version selector in projects such as scylla cloud.

## How to test this PR

1. In ``conf.py``, add the following option under ``html_theme_options``:

```
html_theme_options = {
    "hide_versions_dropdown": "true",
}
```

2. Run ``make multiversionpreview`` and open http://localhost:5500/master/getting-started/. The multiversion selector should not appear.
